package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "dualshock-controller",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
I'm pretty sure that this shouldn't be a part of the PR, as usually maintainers bump the version of the module themselves in a separate commit.